# Lidar Range value from 342°- 18°
Range   Front
# Lidar Range value from 19°- 55°
Range   Front_Left
# Lidar Range value from 56°- 92°
Range   Left
# Lidar Range value from 268°- 304°
Range   Right
# Lidar Range value from 305°- 341°
Range   Front_Right